spec.md

```diff
@@ -3936,6 +3945,11 @@ intmodule = "intmodule" , id , ":" , [ info ] , newline , indent ,
               { "parameter" , "=" , ( int | string_dq ) , newline } ,
             dedent ;
 
+(* Group definitions *)
+defgroup = "defgroup" , id , ":" , [ info ] , newline , indent ,
+             { port , newline } ,
```

Comment:
i dont fully understand this syntax, but shouldn't it have a { defgroup }, here?

Reply:
Yep.